docs/docs/docs/01-core/content-manager/01-documents.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Documents
+description: An introduction to documents in the content-manager
+tags:
+  - content-manager
+  - documents
+---
+
+At the very core of the CM is the concept of a document. Although the logic for creating a document lives primarily within the `@strapi/core` package, we still need to understand what they are to interact with them. Introduced for the _new_ draft & publish feature in V5 (Q1 2024), documents are essentially a matrix of entries, this matrix grows in complexity based on the amount of dimensions your doucment has. For example, having Draft & Publish and Internationalisation (with 2 locales) enabled would give you 4 entries within the document – a draft for each locale and a published entry for each locale.
+
+:::note
+Draft & Publish is optional on content-types
+:::
+
+## Getting Documents
+
+We get documents via the [`useDocument`](./hooks/use-document.mdx) hook, which requires us to understand which "model" we want – the model is basically which content-type e.g. `api::article.article`, the "kind" of content-type e.g. `single-type` or `collection-type` and the specific `id` of said document. Using these three parameters we can fetch any document, without passing specific parameters e.g. `locale` we get a default version defined by the user's application. However, all dimensions should be queryable to narrow down the point on the document matrix to a specific entry e.g. `published & 'en-GB'`.
```

Comment:
Should we say specific `documentId` instead?

Reply:
I'd elect no, only because the `id` in the FE is the `documentId` in the DB only, its just `id` for the FE, unless we start writing a BE explanation.... then we can tweak it?